### ISSUE TITLE
qemu-img executable not found

### ISSUE THREAD

**Issue by JulioPDX** [carlmontanari/boxen#20]
Trying to package a CSR image and running into the following error, any help is appreciated!

```
juliopdx@containerlab:~$ boxen package --disk ~/boxen/csr1000v-universalk9.17.03.02-serial.qcow2 
      info   1645663051 package requested for disk '/home/juliopdx/boxen/csr1000v-universalk9.17.03.02-serial.qcow2'
  critical   1645663051 error copying source disk image: exec: "qemu-img": executable file not found in $PATH

  critical   1645663051 error allocating disks for packaging: exec: "qemu-img": executable file not found in $PATH
        🆘 finished unsuccessfully in 0 seconds 
exec: "qemu-img": executable file not found in $PATH
juliopdx@containerlab:~$ 
```

**Comment by carlmontanari** [comment]
Hey Julio! Thanks for opening this!

I know you already got this sorted from discord chat, but going to put it here for posterity too.

TL;DR for now gotta install qemuimg and/or mkisofs (Darwin) to build the base images. In the near future @hellt pr that gives us a way to make this happen *in* a container (see #14) so you won't need to worry about having those things on your local system (at least for containerizing things -- ofc you will need qemu and stuff to run in "local" mode though!).

Going to close, but feel free to re-open and/or talk on discord!

Carl
